Index.html

```diff
@@ -0,0 +1,322 @@
+            </ul>
+          </div>
+        </nav>
+      </header>
+      <section class="headline">
+        <div id="head-container">
+          <h1 class="head">
+            I'm John.   </br>
+            Glad to see </br>
+            you!
+          </h1>
+          <p class="resume">
+            I’m a software developer! I can help you </br>
+            build a product, feature orwebsite Look </br>
+            through some of my work and </br>
+            experience. If you like what you see and </br>
+            have a project you need coded, don’t </br>
+            hesitate to contact me! 
+          </p>
+          <h2 class="connect">LET’S CONNECT</h2>
+          <ul class="social-media-icons">
+            <li>
+              <a rel="noopener" href="https://twitter.com/jovasal81" target="_blank">
+                <img src="./images/twitter.png" alt="twitter" />
+              </a>
+            </li>
+            <li>
+              <a rel="noopener" href="https://www.linkedin.com/in/john-alonso-vasquez-salcedo-95749632/" target="_blank">
+                <img src="images/Linkedin.png" alt="linkedIn" />
+              </a>
+            </li>
+            <li>
+              <a rel="noopener" href="https://medium.com/@jovasal" target="_blank">
+                <img src="images/medium.png" alt="medium" />
+              </a>
+            </li>
+            <li>
+              <a rel="noopener" href="https://github.com/jovasal" target="_blank">
+                <img src="images/github.png" alt="git" />
+              </a>
+            </li>
+            <li>
+              <a rel="noopener" href="https://wellfound.com/john-vasquez-12" target="_blank">
+                <img src="images/angel.png" alt="angel" />
+              </a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <main class="secpa">
+        <nav class="grid-container">
+          <ul class="flex1">
+            <li class="snapshoot"></li>
+            <li class="square">
+              <li class="primarytext">
+                <h1 class="tonic">Tonic</h1>
+                <div class="cardtonic">
+                  <h2 class="CANOPY">CANOPY</h2>
+                  <img src="./images/Counter.png" alt="counter count" class="dot1" />
+                  <h2 class="BackE">Back End dev</h2>
+                  <img src="./images/Counter.png" alt="counter count" class="dot" />
+                  <h2 class="date">2015</h2>
+                </div>
+                <div clas="adayl2">
+                  <p class="adaylp">
+                    A daily selection of privately </br> 
+                    personalized reads; no accounts or </br> 
+                    sign-ups required.
+                  </p>
+                </div>
+                <div class="botonic">
+                  <h3 class="tonichtml">html</h3>
+                  <h3 class="toniccss">css</h3>
+                  <h3 class="tonicJava">JavaScript</h3>
+                </div>
```

Comment:
- [ ] kindly consider displaying this list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li` as the are list elements and remember its for semantic reasons

Reply:
Changes were made succesfully.

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/745c947e-248b-41d6-a7a2-d11cba901e15)
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/fad0f0bd-a133-443f-a96b-e491debccc20)
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/2b53697d-8194-443f-ae92-dccf35ddd50d)
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/0b067bad-518a-4879-b614-00d3700965bd)